Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
 
 Incidentally, the IETF registration desk is often a convenient place to arrange to meet people. If someone says "meet me at registration", you should clarify if they mean the IETF registration desk, or the hotel registration desk. This has been a common cause of missed connections.
+
 3.2 Take the Plunge and Stay All Week!
 
 IETF WG meetings are scheduled from Monday morning through Friday afternoon. Associated non-WG meetings often take place on the preceding or following weekends. It is best to plan to be present the whole week, to benefit from cross-fertilization between Working Groups and from corridor discussions. As noted below, the agenda is fluid, and there have been many instances of participants missing important sessions due to last-minute scheduling changes after their travel plans were fixed. Being present the whole week is the only way to avoid this annoyance.
 
 If you cannot find meetings all week to interest you, you can still make the most of the IETF meeting by working between sessions. Most IETF attendees carry laptop computers, and it is common to see many of them in the terminal room or in the hallways working during meeting sessions. There is often good wireless Internet coverage in many places of the meeting venue (restaurants, coffee shops, and so on), so catching up on email when not in meetings is a fairly common task for IETFers.
+
 3.3 Newcomer Training
 
-Newcomers are encouraged to attend the Newcomer's Orientation on Sunday afternoon, which is especially designed for first-time attendees. The orientation is organized and conducted by the IETF EDU team and is intended to provide useful introductory information. The session covers what all the dots on name tags mean, the structure of the IETF, and many other essential and enlightening topics for new IETFers.
+Newcomers are encouraged to attend the Newcomer's Tutorial on Sunday afternoon, which is especially designed for first-time attendees. The tutorial is organized and conducted by the IETF EDU team and is intended to provide useful introductory information. The session covers what all the dots on name tags mean, the structure of the IETF, and many other essential and enlightening topics for new IETFers. If you are unable to attend this session, recorded ones from previous meetings are available ([https://www.ietf.org/about/participate/tutorials/]).
+
+Later in the afternoon is the Quick Connections session where newcomers have the chance to get to know senior IETF participants, and ask questions. The Quick Connections session is followed by the Newcomer's Meet and Greet, which is only open to newcomers and WG chairs. This is a great place to try to find people knowledgeable in the areas in which you are interested. Feel free to approach any WG chair, not just those in your area, to either learn about their WG or to have them help find you someone in yours.
```

Comment:
The Quick Connections session is still quite open and people need to register beforehand, so am not sure whether we should put more boundaries here. We are still experimenting with it in EDU.

Reply:
Sorry, to be clear, I'm referring to the "Newcomer's Meet and Greet," not the Quick Connections.

Reply:
As Alissa said, the "Newcomer's Meet and Greet' is probably gonna undergo some changes since newcomers get quite lost and it is mostly a lot of experienced IETF-ers seeing each other in person for the first time in three months and enjoy some beer&wine while chatting with each other. 

So I don't think the problem is that there are too many newcomers. 

Having said that, making changes here, or even removing the Newcomers Meet and Greet would be a loss if we don't know what we'll replace it with.

Reply:
I think it would be useful to be a bit more pushy than "Feel free to approach chairs...".

Maybe:
This is a good opportunity to connect to the chairs of the working group your are interested in. If you can't find the right person easily or don't know which working group your interests are connected to, feel free to approach anybody without a newcomer ribbon to help you to get in touch with the right people.

Reply:
Accepted and changed.